pom.xml

```diff
@@ -50,7 +49,7 @@
 		<equalverifier.version>2.4.1</equalverifier.version>
 		<hibernate.jpa.version>1.0.0.Final</hibernate.jpa.version>
 
-		<jacoco-maven-plugin.version>0.7.9</jacoco-maven-plugin.version>
+		<jacoco-maven-plugin.version>0.8.7</jacoco-maven-plugin.version>
```

Comment:
Not a bad idea to update, but curious if it was required for something in the PR or just "clean up as we go"?

Reply:
> Not a bad idea to update, but curious if it was required for something in the PR or just "clean up as we go"?

Jacoco was failing with a strange error.